post_database.php

```diff
@@ -0,0 +1,60 @@
+<?php
+	include('config/config.php');
+	include('include/db_query.php');
+
+	//InsertBlogPost('Amanda Hua', '2018-05-25', 'TestPost', 'Hi this is a test post');
+	$post = GetBlogPost($_REQUEST['postID']);
+	//var_Dump($post);
+	DisplayPost($post);
+
+
+	function InsertBlogPost($author, $date, $title, $body){
```

Comment:
Same comment about putting this in /include. It's normally a red flag when function definitions are mixed with code that calls the functions (lines 6-8) because it means there's no way to include these functions without also running the code above.